identity/src/main/java/com/stripe/android/identity/IdentityOnBackPressedHandler.kt

```diff
@@ -14,14 +14,17 @@ import com.stripe.android.identity.navigation.clearDataAndNavigateUp
 import com.stripe.android.identity.navigation.routeToScreenName
 import com.stripe.android.identity.networking.models.VerificationPage.Companion.requireSelfie
 import com.stripe.android.identity.viewmodel.IdentityViewModel
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 /**
  * Handles back button behavior based on current navigation status.
  */
 internal class IdentityOnBackPressedHandler(
     private val verificationFlowFinishable: VerificationFlowFinishable,
     private val navController: NavController,
-    private val identityViewModel: IdentityViewModel
+    private val identityViewModel: IdentityViewModel,
+    private val coroutineScope: CoroutineScope
```

Comment:
with the suggestion above you might be able to avoid passing the lifecycle scope around.